app/views/audit_mailer/summary.text.erb

```diff
@@ -0,0 +1,15 @@
+<%= _("Foreman audit summary") %>
+=================================
+<% if @audits.size > 0 %>
+  <%= n_('Displaying %{num_audits} of %{total_audits} audits',
```

Comment:
singular would be `%{total_audits} audit` now you've changed what variable n_ is based on